docs/statistics/sample-size-calculator/usage.md

```diff
@@ -4,35 +4,71 @@ sidebar_position: 2
+:::tip
+Often it is desirable to look at more than one metric at the same time. To determine the minimum runtime that satisfies requirements on multiple metrics, look for the column where _all_ of the MDEs are smaller than the required MDEs.
 
-![Sample size calculator example](/img/planning-experiments/sample_size_calculator_example.png)
+The required MDE is not necessarily the same for all metrics: for example, you may desire a smaller MDE for a critical guardrail metric, to have a tighter control on degradations. Adding more metrics will either increase the minimum runtime or leave it unchanged, but never decrease it.
+:::
 
-In statistical terms, [Minimum Detectable Effect (MDE)](/statistics/sample-size-calculator/mde) is defined as the smallest lift that can be detected a certain percent of the time, while ensuring that the experiment's confidence intervals contain the true value a specified percent of the time; these two numbers (called Power and Confidence Level) can be configured under Advanced Settings, on the left. You may notice while using those controls that higher power and confidence levels correspond to larger Minimum Detectable Effects for all metrics.
+Below the main table interface, you will find a natural-language presentation of the same data. This interface may be useful for explaining the MDE calculation to others. The data used for this presentation is the same as that used in the table above.
 
-Often it is desirable to compute MDEs for several metrics at once to ensure that several metrics of interest will achieve a given MDE. You can add more than one metric to the table using the _Add..._ control in the top-right corner. You can remove a metric from the table using the Actions (...) control next to each metric name.
+## Adding or removing metrics
 
-Below the main table interface, you will find a natural-language presentation of the same data. This interface may be useful for explaining the MDE calculation to others.
+Unlike settings, metrics cannot be changed after you run the sample size calculation. Adding or removing metrics requires creating and launching a new calculation. You can do so by clicking the _Duplicate_ button that appears for completed calculations. This will create a new calculation with all the parameters copied over. You can then add or remove metrics as needed, and finally click _Run Calculation_.
 
-Once you have determined the experiment runtime that will achieve the desired MDE, you can close the interface and proceed to set up a new experiment.
+:::tip
+If a previous calculation for the same entry point already contains all the metrics you are interested in, you can save time and compute by re-using that saved calculation (accessible by URL). Note that this may include metrics that are not relevant to you; and all analysis settings are shared. If you need a calculation that only includes the metrics of interest, you need to create a new calculation.
```

Comment:
This "tip" is borderline insulting (and almost creates more confusion than not having it, IMO), but I added it per ShareChat's request. cc @Ceh265

Reply:
Yes this seems fairly obvious


> If you need a calculation that only includes the metrics of interest, you need to create a new calculation.

Imo we could get rid of this line